src/components/StakingPosition/StakingPosition.tsx

```diff
@@ -120,6 +121,15 @@ const StakingPosition: React.FC<Props> = ({ id, usdcBalance, sherRewards, lockup
             </Column>
           </Row>
         )}
+        {id.lte(275) && (
```

Comment:
id <= 275 && restake_count == 0? Otherwise the message will show if it has been restake